test/common/versionhelper.js

```diff
@@ -0,0 +1,26 @@
+'use strict'
+
+module.exports = {
+  'getSQLServerVersion' (sql, done) {
+    const req = new sql.Request()
+    return req.query("select SERVERPROPERTY('productversion') as version").then(result => {
+      return result.recordset[0].version
+    }).catch(done)
```

Comment:
I don't think we need to or should be passing around the done callback. Just let this promise reject and the parent promise can catch it.